src/main/java/be/syntra/devshop/DevshopFront/controllers/UserController.java

```diff
@@ -21,28 +30,53 @@ public UserController(CartService cartService) {
         this.cartService = cartService;
     }
 
-    @GetMapping("/cart/overview")
+    @GetMapping("/cart")
+    public String displayCartWhenLoggedIn() {
+        log.info("displayCartWhenLoggedIn()");
+        return REDIRECT_CART_DETAIL_URL;
+    }
+
+    @GetMapping("/cart/detail")
     public String displayCartOverview(Model model) {
         log.info("displayCartOverview()");
         model.addAttribute("cart", cartService.getCart());
```

Comment:
extract: 
model.addAttribute("cart", cartService.getCart());
        PaymentDto paymentDto = PaymentDto.builder()
                .totalCartPrice(cartService.getCartTotalPrice())
                .paymentOptions(Arrays.asList(PaymentOption.values()))
                .build();
        model.addAttribute(PAYMENT, paymentDto);
It is used twice in this controller. IntelliJ didn't pick it up, because the second time, some lines have switched position

Reply:
when extracted, the static final for PAYMENT becomes redundant as well